runtime/cudaq/platform/default/rest_server/helpers/RestRemoteServer.cpp

```diff
@@ -120,6 +120,10 @@ class RemoteRestRuntimeServer : public cudaq::RemoteRuntimeServer {
       : cudaq::RemoteRuntimeServer(),
         m_simHandle(DEFAULT_NVQIR_SIMULATION_BACKEND,
                     loadNvqirSimLib(DEFAULT_NVQIR_SIMULATION_BACKEND)) {}
+
+  virtual int version() const override {
+    return cudaq::RestRequest::REST_PAYLOAD_VERSION;
```

Comment:
(Hopefully) one last request - can we please allow an environment variable override for this? It might be useful for testing purposes to have the client target different `REST_PAYLOAD_VERSION` values other than what it was built with. I.e. In future integration and/or backwards-compatibility testing, we might have client version `3` run tests with server versions `2` and `4` for specific payloads. This wouldn't help if we actually change the API schema, but it might help for cases when the server is bumping the version to signify support for a new runtime function.

Reply:
Please take a look at https://github.com/NVIDIA/cuda-quantum/pull/1328/commits/3c8b12007b4eee99aa92940761ff7eb12bfa6516 and let me know if I missed anything :) 

Basically, both the client and server interfaces have the abstract `version` method. I only added environment variable override for the client side as suggested.

Reply:
Looks great...thanks!